EZP-31236 Add Tooltips as new UI component (#969)
ezsystems/developer-documentation: docs/guide/extending/extending_back_office.md

        arguments:
            $shortDateTimeFormatter: '@ezplatform.user.settings.short_datetime_format.formatter'
```

### Add tooltips

Tooltips help provide more information for the users when they hover over, focus on, or tap an element.

> eZ Platform tooltips are using [Bootstrap tooltips.](https://getbootstrap.com/docs/4.1/components/tooltips/)

Reviewer: ```suggestion
eZ Platform tooltips use [Bootstrap tooltips.](https://getbootstrap.com/docs/4.1/components/tooltips/)
```

> To create a tooltip you have to add a `title` attribute with  custom tooltip text for HTMLnode.

Reviewer: ```suggestion
To create a tooltip you have to add a `title` attribute with a custom tooltip text for HTMLnode.
```

```html
<button title="custom tooltip text"</button>
```

Additionally, you can add following attributes:

- `data-extra-classes` - an additional class for tooltip container `.tooltip`
- `data-tooltip-container-selector` - a css selector for a tooltip container (Bootstrap tooltip `data-container` attribute)

Example of a tooltip with additional attributes:

```html
<button title="custom tooltip text" data-extra-classes="additinal_class" data-tooltip-container-selector="selector">click me</button>
```

> You can also add tooltips helpers to JavaScript `eZ.helpers` object:

Reviewer: ```suggestion
You can also add tooltip helpers to the JavaScript `eZ.helpers` object:
```
